scraper/newsscrapper/settings.py

```diff
@@ -1,97 +1,32 @@
-# Scrapy settings for newsscrapper project
-#
-# For simplicity, this file contains only settings considered important or
-# commonly used. You can find more settings consulting the documentation:
-#
-#     https://docs.scrapy.org/en/latest/topics/settings.html
-#     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
-#     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+#--------------------------------------------------------------------------
+# Your variables from .env file
+PORT_NUMBER = os.getenv("PORT_NUMBER")
+#--------------------------------------------------------------------------
 
 BOT_NAME = "newsscrapper"
 
 SPIDER_MODULES = ["newsscrapper.spiders"]
 NEWSPIDER_MODULE = "newsscrapper.spiders"
+ITEM_PIPELINES = {
+   "newsscrapper.pipelines.NewsscrapperPipeline": 300,
+}
 
-
-# Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = "newsscrapper (+http://www.yourdomain.com)"
-
-# Obey robots.txt rules
 ROBOTSTXT_OBEY = True
 
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
 CONCURRENT_REQUESTS = 16
 
```

Comment:
add `LOG_LEVEL = 'WARNING' to suppress logging levels lower than warning ( debug, info, etc. )

Reply:
@AardhynLavender, I have tried this before submitting this PR, but it didn't work. I found a different solution that works

Reply:
great!